tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1344,6 +1392,36 @@ remove_dead_instrs(void)
   }
 }
 
+/*
+ * Check if the break instruction is having a loop pragma
```

Comment:
break or the branch instruction (back edge)?

Reply:
Renamed to `branch`.